services/scripts.py

```diff
@@ -1,15 +1,49 @@
 # Music recommendation application
 import csv
-from .cfg import ARTIST_INDEX, LIKES_INDEX
+
+from cfg import TableEnum
+
+
+class Song:
+    def __init__(self, row: list[str]) -> None:
+        self.id = int(row[TableEnum.ID.value])
+        self.name = row[TableEnum.Track.value]
+        self.album = row[TableEnum.Album.value]
+        self.artist = row[TableEnum.Artist.value]
+
+
+class DataBase:
+    def __init__(self, filename):
+        self.db = {}
+
+        with open(filename, encoding='utf-8-sig') as file:
+            _ = file.readline()
+            reader = csv.reader(file)
+
+            for row in reader:
+                self.add_song(Song(row))
+
+    def add_song(self, song: Song) -> None:
+        self.db[song.id] = song
+
+    def get_song(self, song_id: int) -> Song:
+        return self.db.get(song_id)
+
+
+if __name__ == '__main__':
+    db = DataBase(filename='../data/Spotify_Youtube.csv')
+
+    print(db.get_song(1).name)
```

Comment:
No check added that get object did not return a None obj, in case it does, you will get an error saying something like NoneType object has no attr name.

Reply:
> No check added that get object did not return a None obj, in case it does, you will get an error saying something like NoneType object has no attr name.

returning to the case with `get_song(…) -> None`, we can create an empty version of the `Song` class, as an example:
```py
>>> empty_song = Song()
>>> empty_song.name
None
```

Reply:
but this is just a suggestion